GWB_Green_Domain_Call.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Create a Green Web Foundation Green Domain GET to get the data from the Database
+ * Details: https://datasets.thegreenwebfoundation.org/daily_snapshot/greendomain
+ */
+function newGWFRequest(url="web.dev.com"){
```

Comment:
It's not clear why a default value for the url parameter is included. It seems like this should return null or throw if there is no URL passed.

Reply:
Oh yes! I wrote newGWFRequest(url="web.dev.com") to test things locally (I can't newGWFRequest from the Apps Script UI with parameters)
I removed it, added a test clause and a demo function to test things out smoothly:

```
/**
 * A simple demo function to verify from GWB_Green_Domain_Call.gs that we are getting the expected answer.
 * Logging the content locally using Logger.log to verify that it is matching expectations/
 * Result should be {url=https://api.thegreenwebfoundation.org/api/v3/greencheck/web.dev.com, muteHttpExceptions=true}
 */
function newGWFRequestDemo(){
  const url = "web.dev.com";
  let request = newGWFRequest(url)
  Logger.log(request);
}
```